examples/snapshot-metadata-lister/main.go

```diff
@@ -155,6 +168,34 @@ func main() {
 		os.Exit(1)
 	}
 
+	if !verify {
```

Comment:
I kind-of missed the fact that this verify is happening as a _secondary step_ of the existing lister command.  Nothing wrong with that, but why make 2 passes?  Why not just use the verify emitter when the `--verify` flag is set?